lib/basedriver/driver.js

```diff
@@ -470,10 +470,11 @@ class BaseDriver extends Protocol {
    * should be made available to overridden methods.
    * @param {string} method - HTTP method of the route
    * @param {string} url - url of the route
+   * @param {object} body - webdriver request body
```

Comment:
I assume the type of body should be something like `{?*}` or it is always an object?

Reply:
yeah good point